soc/ti/simplelink/cc23x0/Kconfig.defconfig

```diff
@@ -10,8 +10,20 @@ if SOC_SERIES_CC23X0
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	default $(dt_node_int_prop_int,/cpus/cpu@0,clock-frequency)
 
-config SYS_CLOCK_TICKS_PER_SEC
-	default 1000
+if CC23X0_RTC_TIMER
+	config SYS_CLOCK_HW_CYCLES_PER_SEC
+		default 32768
+
+	config SYS_CLOCK_TICKS_PER_SEC
+		default 125375
```

Comment:
Where does this value come from? It seems unusually large.

Reply:
It was calculated , it is not a magic number. It is result from ticks which sys clock is making calculated to correct resolution

Reply:
Could you share how it was calculated in the commit message or as a comment?